internal/resources/slo/resource_slo.go

```diff
@@ -0,0 +1,477 @@
+													Required: true,
+												},
+												"value": &schema.Schema{
+													Type:     schema.TypeString,
+													Required: true,
+												},
+											},
+										},
+									},
+									"annotations": &schema.Schema{
+										Type:        schema.TypeList,
+										Optional:    true,
+										Description: "Annotations to attach only to Fast Burn alerts.",
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"key": &schema.Schema{
+													Type:     schema.TypeString,
+													Required: true,
+												},
+												"value": &schema.Schema{
+													Type:     schema.TypeString,
+													Required: true,
+												},
+											},
+										},
+									},
+								},
+							},
+						},
+						"slowburn": &schema.Schema{
+							Type:        schema.TypeList,
+							Optional:    true,
+							Description: "Alerting Rules generated for Slow Burn alerts",
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"labels": &schema.Schema{
+										Type:        schema.TypeList,
+										Optional:    true,
+										Description: "Labels to attach only to Slow Burn alerts.",
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"key": &schema.Schema{
+													Type:     schema.TypeString,
+													Required: true,
+												},
+												"value": &schema.Schema{
+													Type:     schema.TypeString,
+													Required: true,
+												},
+											},
+										},
+									},
+									"annotations": &schema.Schema{
+										Type:        schema.TypeList,
+										Optional:    true,
+										Description: "Annotations to attach only to Slow Burn alerts.",
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"key": &schema.Schema{
+													Type:     schema.TypeString,
+													Required: true,
+												},
+												"value": &schema.Schema{
+													Type:     schema.TypeString,
+													Required: true,
+												},
+											},
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func resourceSloCreate(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	var diags diag.Diagnostics
+
+	slo, err := packSloResource(d)
+	if err != nil {
+		diags = append(diags, diag.Diagnostic{
+			Severity: diag.Error,
+			Summary:  "Unable to Pack SLO",
+			Detail:   err.Error(),
+		})
+		return diags
+	}
+
+	client := m.(*common.Client).GrafanaAPI
+	response, err := client.CreateSlo(slo)
+
+	if err != nil {
+		diags = append(diags, diag.Diagnostic{
+			Severity: diag.Error,
+			Summary:  "Unable to create SLO - API",
+			Detail:   fmt.Sprintf("API Error Message:%s", err.Error()),
+		})
+		return diags
+	}
+
+	d.SetId(response.UUID)
+	resourceSloRead(ctx, d, m)
+
+	return diags
```

Comment:
The read error is being ignored
```suggestion
	return resourceSloRead(ctx, d, m)
```